File: R/8kyu/8kyu-convert-a-number-to-a-string.r
# Convert a Number to a String!
# Difficulty: 8 kyu

number_to_string <- function(n) {
  as.character(n)
}


# Tests
test_that("fixed tests", {
  expect_equal(number_to_string(67), '67')
  expect_equal(number_to_string(79585), '79585')
  expect_equal(number_to_string(3), '3')
  expect_equal(number_to_string(-1), '-1')
})

File: Julia/8kyu/8kyu-convert-a-number-to-a-string.jl
# Convert a Number to a String!
# Language: Julia
# Version: 1.5
# Difficulty: 8 kyu

function numbertostring(num)
    string(num)
end


# Tests
using FactCheck

facts("Fixed Tests") do
  @fact numbertostring(67) --> "67"
  @fact numbertostring(79585) --> "79585"
  @fact numbertostring(79585) --> not(79585)
  @fact numbertostring(1+2) --> "3"
  @fact numbertostring(1-2) --> "-1"
end